samples/QuickBooks.gs

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
Please add a top level comment that better sets the stage about the entire sample. Here's an example:

https://github.com/gsuitedevs/apps-script-oauth2/blob/master/samples/AdobeSign.gs#L1

Reply:
Done!